static/app/components/profiling/flamegraph/flamegraph.tsx

```diff
@@ -309,6 +320,26 @@ function Flamegraph(): ReactElement {
     hasUIFrames,
   ]);
 
+  const batteryChart = useMemo(() => {
+    if (!hasCPUChart) {
+      return LOADING_OR_FALLBACK_CPU_CHART;
+    }
```

Comment:
`hasBatteryChart` and `LOADING_OR_FALL_BACK_BATTERY_CHART?

Reply:
Too fast copy pasta, nice find